tb_plugin/torch_tb_profiler/profiler/run_generator.py

```diff
@@ -306,48 +307,73 @@ def __init__(self, all_profile_data):
     def generate_run_profile(self):
         profile_run = DistributedRunProfile()
         profile_run.views.append(consts.DISTRIBUTED_VIEW)
+        profile_run.gpu_info = self._generate_gpu_info()
         profile_run.steps_to_overlap = self._generate_overlap_graph()
         profile_run.steps_to_wait = self._generate_wait_graph()
         profile_run.comm_ops = self._generate_ops_table()
         return profile_run
 
+    def _generate_gpu_info(self):
+        result = OrderedDict()
+        result["metadata"] = {"title": "Device Info"}
```

Comment:
add logic to read the data from trace file.

Reply:
done